scene/2d/sprite_2d.cpp

```diff
@@ -272,7 +275,21 @@ int Sprite2D::get_vframes() const {
 
 void Sprite2D::set_hframes(int p_amount) {
 	ERR_FAIL_COND_MSG(p_amount < 1, "Amount of hframes cannot be smaller than 1.");
+	if (vframes > 1) {
+		// recalculate frame
```

Comment:
```suggestion
		// Adjust the frame to fit new sheet dimensions.
```

The rephrasing is secondary, but comment needs to start with a capital letter and end with a period.